s4e-web/src/app/views/settings/manage-institutions/add-institution/add-institution.component.html

```diff
@@ -0,0 +1,158 @@
+<div class="breadcrumb">
+  <span i18n>Tutaj jesteś: </span>
+  <span>
+    <a routerLink="/settings" i18n>Start</a> >
+  </span>
+  <span>
+    <a routerLink="manage-institutions" i18n>Zarzadzaj instytucjami</a> >
+  </span>
+  <span i18n>Dodaj instytucję</span>
+</div>
+
+<br><br>
+<h1 i18n>Dodaj instytucję</h1>
+<br><br>
+
+<form class="form informational-form" [formGroup]="form">
+  <div class="alert alert-danger" *ngIf="error$ | async" data-e2e="alert-message">
+    <ng-container [ngSwitch]="(error$ | async).status">
+      <ng-container *ngSwitchCase="400" i18n>Niepoprawnie wypełniony formularz</ng-container>
+      <ng-container *ngSwitchCase="500" i18n>Wystąpił błąd serwera</ng-container>
+      <ng-container *ngSwitchDefault i18n>Nieoczekiwany błąd</ng-container>
+    </ng-container>
+  </div>
+
+  <div class="alert alert-danger" *ngIf="form.errors" data-e2e="alert-message-details">
+    <ul>
+      <li *ngFor="let error of (form.errors | errorKeys)" [ngSwitch]="error">
+        <ng-container *ngSwitchDefault i18n>Nieoczekiwany błąd</ng-container>
+      </li>
+    </ul>
+  </div>
+
+  <p><b>Instytucja nadrzędna (rodzic)</b></p>
+  <p>Klikając w poniższe pole można zmienić dla nowo tworzonej instytucji jednostkę narzędną.</p>
```

Comment:
Shouldn't this and other text elements in here have the i18n attribute as well? Some have it, so it seems a little inconsistent to me